.github/workflows/backport-dispatch.yaml

```diff
@@ -0,0 +1,34 @@
+name: Automated Backporting via Dispatch
+
+on:
+  workflow_dispatch:
+    inputs:
+      targetBranch:
+        description: 'Release (22.0, 24.0, ...)'
+        required: true
+        type: string
+        default: "24.0"
+      pullRequest:
+        description: 'Pull request (just the ID)'
+        required: true
+        type: string
+      dryRun:
+        description: 'Dry run'
+        required: false
+        default: false
+        type: boolean
+
+jobs:
+  backporting:
+    name: "Backporting"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Backporting
+        uses: kiegroup/git-backporting@bce5dd4f9969d47da9cbb6ed06abbf87216afc98 # 4.5.1
```

Comment:
For consistency, should we not be using the [existing backport script](https://github.com/keycloak/keycloak/blob/ceed62213df586cc7fe0d3ef45e7b11057ded3d2/.github/scripts/pr-backport.sh) here directly? If we're going to use an action going forward I'd argue it should be the only way to do a backport to avoid inconsistencies.

Reply:
Personally I would just use the script and not an action, as I find it easy and quick to do that, but if others prefer a action then I don't see a big deal with having both. Good point about using the same approach both places thouyy